# *Ralstonia* Wilt Dashboard & *Ralstonia solanacearum* species complex (RSSC) Global Diversity Database

[![DOI](https://zenodo.org/badge/DOI/10.5281/zenodo.16945323.svg)](https://doi.org/10.5281/zenodo.16945323)

## Background
Welcome to the *Ralstonia* Wilt Dashboard, a comprehensive tool designed to explore and analyze the genetic diversity of *Ralstonia solanacearum* species complex (RSSC). The *Ralstonia solanacearum* species complex (RSSC) are a diverse group of soil and waterborne pathogens that causes wilt and rot diseases of many economically important plant species. The species complex has been sub-divided into three distinct genomospecies: *R. solanacearum*, *R. pseudosolanacearum*, and *R. syzygii*. Different RSSC lineages emerged and diversified on distinct continents: *R. solanacearum* in the Americas, *R. syzygii* in the SE Asian archipelagoes (e.g. Indonesia), the phylotype I branch of *R. pseudosolanacearum* in continental Asia, and the phylotype III branch of *R. pseudosolanacearum* in Africa.
RSSC are broad-host range, generalist pathogens where individual strains have complex patterns of host range. Because most lineages that infect economically important crop plants can also cause latent infections of ornamental plants, latent infections of clonally propagated ornamental plants facilitate the transmission of exotic RSSC to new countries. The *Ralstonia* Wilt Dashboard integrates data from hundreds of sources. Please read our companion pre-print on BioRxiv for more details: https://www.biorxiv.org/content/10.1101/2020.07.13.189936

## Purpose
- Communicate public data to stakeholders through a dashboard that visualizes geographic distribution patterns of *Ralstonia solanacearum* species complex (RSSC) lineages and patterns of *Ralstonia solanacearum* species complex (RSSC) host range 
- To build a framework for understanding how and where exotic RSSC lineages can enter, establish, and spread in the U.S. 
- To develop an interactive dashboard that provides user-friendly access to host, location, and other already collected pathogen metadata on thousands of RSSC strains

## Goals
- This interactive dashboard visualizes large and complex RSSC data in interactive maps, charts, and tables for stakeholders to explore.
- Maps will highlight the geographic distribution of isolated RSSC strains by phylotype. The interactive features will allow stakeholders to filter plotted data by taxonomic classification (phylotypes or sequevars), by host (plant species or botanical families), and by geographic location.
- Charts will visualize phylotype abundance by host and location.

## Reporting Errors or Bugs
- Any issues, questions, or comments about the dashboard can be directed to the *Ralstonia*-dashboard issue tracker on our github or to the Google form linked at the top of the dashboard webpage.

## About the Team 
- This dashboard is a collaboration between the [Lowe-Power Lab at UC Davis](https://lowepowerlab.ucdavis.edu) and the Data Intensive Biology lab at UC Davis and is under guidance by **Dr. Tessa Pierce-Ward** and **Dr. Tiffany Lowe-Power**.
- Version 1 of the *Ralstonia* Wilt Dashboard was developed by **Vienna Elmgreen** with geocoding done by **Benjamin Ramirez**.
- Assistance with geocoding was provided by **Rituraj Sharma** and **Dr. Boris Vinatzer** (Virginia Tech).

## Acknowledgments
- We gratefully acknowledge the global plant pathology community for investigating and reporting *Ralstonia* incidence and isolation information. We are grateful to [Dr. Emerson Del Ponte](https://emersondelponte.netlify.app/) for sharing advice and code from the [*Fusarium graminearum* species complex Db](https://edelponte.shinyapps.io/FGSCdb/).
- Support was provided by the U.S. Department of Agriculture Hatch Projects (#1023861) and Pests and Beneficial Species in Agricultural Production Systems (A1112) program, project award no. 2024-67013-42781, from the U.S. Department of Agriculture’s National Institute of Food and Agriculture. Additionally, this material was made possible, in part, by a PPA7721 Cooperative Agreement from the United States Department of Agriculture’s Animal and Plant Health Inspection Service (APHIS). It may not necessarily express APHIS’ views.

## Related Databases
There are several related databases that users might find helpful:
- **EPPO (European Plant Protection Organization) databases** for [*Ralstonia solanacearum*](https://gd.eppo.int/taxon/RALSSL), [*Ralstonia pseudosolanacearum*](https://gd.eppo.int/taxon/RALSPS), and  [*Ralstonia syzygii*](https://gd.eppo.int/taxon/RALSSY/distribution).
- [**Ralstonia PhytobacExplorer**](https://phytobacexplorer.warwick.ac.uk/species/index/ralstonia) A database for analyzing and visualizing genomic variation within plant-pathogenic bacteria.
- [**RalstoT3E:**](https://iant.toulouse.inra.fr/bacteria/annotation/site/prj/T3Ev3/) A database dedicated to *Ralstonia* Type III effectors, which influence the cryptic patterns of host range in the RSSC.
- [**The *Ralstonia* Phylogenomics KBase Narrative**](https://narrative.kbase.us/narrative/189849)(requires a free kbase.us account). A graphical user interface for phylogenomics and comparative genomes that is pre-populated with a curated and comprehensive list of public RSSC genomes.
- **CABI Compendium Entries** for [*Ralstonia solanacearum*](https://www.cabidigitallibrary.org/doi/full/10.1079/cabicompendium.45009) and [*Ralstonia pseudosolanacearum*](https://www.cabidigitallibrary.org/doi/abs/10.1079/cabicompendium.15285723).

## Citation
- For the data, please read/cite our companion pre-print on BioRxiv: https://www.biorxiv.org/content/10.1101/2020.07.13.189936
- To cite the web dashboard:
   
   Vienna Elmgreen, Tessa Pierce Ward, Benjamin Ramirez, & Tiffany Lowe-Power. (2025). lowepowerlab/dashboard: v1.0 (v1.0). Zenodo. https://doi.org/10.5281/zenodo.16945323
