# *Ralstonia solanacearum* species complex (RSSC) Global Diversity Database

## Background
The *Ralstonia solanacearum* species complex (RSSC) are a diverse group of soil and waterborne pathogens that causes wilt and rot diseases of many economically important plant species. The species complex has been sub-divided into three distinct genomospecies: *R. solanacearum*, *R. pseudosolanacearum*, and *R. syzygii*. Different RSSC lineages emerged and diversified on distinct continents: *R. solanacearum* in the Americas, *R. syzygii* in the SE Asian archipelagoes (e.g. Indonesia), the phylotype I branch of *R. pseudosolanacearum* in continental Asia, and the phylotype III branch of *R. pseudosolanacearum* in Africa.
RSSC are broad-host range, generalist pathogens where individual strains have complex patterns of host range. Because most lineages that infect economically important crop plants can also cause latent infections of ornamental plants, latent infections of clonally propagated ornamental plants facilitate the transmission of exotic RSSC to new countries.

## Purpose
- To leverage public data to analyze geographic distribution patterns of RSSC lineages, including the SA R3B2, with different host ranges
- To communicate these patterns and make this data easily accessible to APHIS and other stakeholders
- To build a framework for understanding how and where exotic RSSC lineages can enter, establish, and spread in the U.S. 
- To develop an interactive dashboard that provides user-friendly access to host, location, and other already collected pathogen metadata on nearly 9,000 RSSC strains
- To use Branchwater to mine public metagenomes for sequences matching RSSC lineages, including the SA R3B2, and make this additional information also accessible through the RSSC Dashboard

## Goals
- This interactive dashboard summarizes RSSC data in accessible maps, charts, and tables for stakeholders to explore.
- Maps will highlight the geographic distribution of isolated RSSC strains through scatter plot maps with proportional circles. The interactive features will allow stakeholders to color strain data points by taxonomic classification (phylotypes or sequevars) or by host (plant species or botanical families) and to filter strain data points by year and geographic location.
- Charts will visualize strain data in stacked bar charts and pie charts to display phylotype and host range data. 
- Tables will display strain data and the interactive features will allow stakeholders to search and filter data to desired specifications.
- Vegetatively propagated plant tissues are the main risk for long-distance movement of RSSC. We will have a tab that focuses on vegetatively propagated crops (potato, Musa sp. [banana/plantain], Zingiber rhizomes [ginger, turmeric], and ornamentals [e.g. geranium, pothos, anthurium, rose, etc.]).

## Resources
- Our RSSC database data is publicly available: https://www.biorxiv.org/content/10.1101/2020.07.13.189936v4.full. 
- Our RSSC Dashboard will be modeled on the Fusarium graminearum species complex database (FGSCdb), which is built from R packages and hosted on a website as a ShinyApp: https://edelponte.shinyapps.io/FGSCdb/.

